src/style.css

```
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+  margin-top: 0;
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 3px;
+  outline: none;
+}
+
+.btn {
+  background: none;
+  border: none;
+  position: absolute;
+}
+
+.big-input {
+  height: 70px;
+  padding: 25px 0 23px 62px;
+  font-weight: 300;
+}
+
+.big-btn {
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  cursor: pointer;
+}
+
+.big-input::placeholder {
+  color: #3d4e61;
+  height: 20px;
+  width: 129px;
+  font-size: 16px;
+  line-height: 21.86px;
+}
```

You don't need to copy-paste styles from figma. Most of them aren't needed here
```suggestion
.big-input::placeholder {
  font-size: 16px;
  line-height: 22px;
}
```

---

Fix that for small form, as well